actix-http/src/lib.rs

```diff
@@ -81,3 +80,36 @@ pub enum Protocol {
 }
 
 type ConnectCallback<IO> = dyn Fn(&IO, &mut Extensions);
+
+// container for data that extract with ConnectCallback
```

Comment:
use doccomments in this file

```suggestion
/// Container for data that extract with ConnectCallback.
```